packages/trader/src/Assets/Trading/Categories/trade-categories-gif.tsx

```diff
@@ -46,7 +45,7 @@ const TradeCategoriesGIF = ({
         case TRADE_TYPES.MATCH_DIFF:
             return <ImageMatchDiff />;
         case TRADE_TYPES.MULTIPLIER:
-            return <ImageMultiplier />;
+            return <ContractTypeDescriptionVideo selected_contract_type='multiplier' />;
```

Comment:
```suggestion
            return <ContractTypeDescriptionVideo selected_contract_type=TRADE_TYPES.MULTIPLIER />;
```
Would this work?

Reply:
Yep, thank you, that should work

Reply:
Oh, I'm stupid, I can just pass selected_contract_type={selected_contract_type}